modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -367,8 +432,48 @@ private NotificationTemplate _addNotificationTemplate(
 					createNotificationRecipientSetting("to", to)),
 				ListUtil.toString(
 					getTermNames(), StringPool.BLANK, StringPool.SEMICOLON),
-				NotificationConstants.TYPE_EMAIL,
-				Collections.singletonList(objectField.getObjectFieldId())));
+				NotificationConstants.TYPE_EMAIL, objectId));
+	}
+
+	private void _addObjectEntrybyEntryManager(String scope, long fileEntryId)
+		throws Exception {
+
+		if (scope == ObjectDefinitionConstants.SCOPE_SITE) {
+			objectEntryManager.addObjectEntry(
+				dtoConverterContext, scopeSiteObjectDefinition,
+				new ObjectEntry() {
+					{
+						properties = scopeSiteObjectEntryValues;
+					}
+				},
+				String.valueOf(_group.getGroupId()));
+		}
+		else {
+			ObjectEntry objectEntry = objectEntryManager.addObjectEntry(
+				dtoConverterContext, parentObjectDefinition,
+				new ObjectEntry() {
+					{
+						properties = parentObjectEntryValues;
+					}
+				},
+				ObjectDefinitionConstants.SCOPE_COMPANY);
+
+			objectEntryManager.addObjectEntry(
+				dtoConverterContext, childObjectDefinition,
+				new ObjectEntry() {
+					{
+						properties = HashMapBuilder.putAll(
+							childObjectEntryValues
+						).put(
+							getObjectRelationshipObjectField2Name(),
+							objectEntry.getId()
+						).put(
+							"attachmentObjectField", fileEntryId
+						).build();
+					}
+				},
+				ObjectDefinitionConstants.SCOPE_COMPANY);
+		}
```

Comment:
Hey @rijema in this case this method is not necessary, we can move this conditional to the method where it's being called.

Thank you!